friendly-captcha/modules/html-forms/html-forms.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+add_filter( 'hf_form_html', function( $html ) {
+    $plugin = FriendlyCaptcha_Plugin::$instance;
+    if (!$plugin->is_configured() or !$plugin->get_html_forms_active()) {
+        return;
+    }
+
+    frcaptcha_enqueue_widget_scripts();
+
+    $widget = frcaptcha_generate_widget_tag_from_plugin($plugin);
+    $html = str_replace( '</form>', $widget . '</form>', $html );
+	return $html;
+});
```

Comment:
This is a bit of hack, but I didn't find a better way to add the widget to the form ...